### PR TITLE
Fix split tracks. Relies on Frederikam/Lavalink#13

### PR DESCRIPTION
**Pull request by napstr**
Todo:

- [x] Update Lavalink to include this PR Frederikam/Lavalink#13

### CODE REVIEW

**Comment by napstr** [comment]
Build will be automagically fixed when Lavalink is updated.

**Comment by Frederikam** [comment]
Lavalink PR was merged, just need to change the version here

**Comment by napstr** [comment]
How are we doing that currently? I was under the impression you would just bump a lavalink number.

**Comment by Frederikam** [comment]
Hello Twitch!

**Comment by Frederikam** [comment]
Just change the commit hash in the FredBoat POM

**Comment by napstr** [comment]
Aight, forgot we're using jitpack, as I'm running local snapshot builds.

File: FredBoat/src/main/java/fredboat/audio/queue/SplitAudioTrackContext.java [hunk 1]
```diff
@@ -45,6 +46,11 @@ public SplitAudioTrackContext(AudioTrack at, long userId, long guildId, long sta
         this.startPos = startPos;
         this.endPos = endPos;
         this.title = title;
+
+
+        // TODO introduce TrackData to FredBoat instead of the AudioTrackContext and SplitAudioTrackContext?
+        // TODO doing that would mean Yet Another Rewrite:tm: of the persistent tracklists
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/audio/queue/SplitAudioTrackContext.java]: Not really necessary. AudioTrackContext is more flexible IMO

Comment (napstr) [reply FredBoat/src/main/java/fredboat/audio/queue/SplitAudioTrackContext.java]: Indeed. Also feels like spilling FredBoat implementation details into Lavalink. Should have removed that comment as I did with a similar one. Will do in a sec.

Comment (Frederikam) [reply FredBoat/src/main/java/fredboat/audio/queue/SplitAudioTrackContext.java]: Also remember to remove `@Deprecated`

**Pull request merged**